jmix-quartz/quartz-flowui/src/main/java/io/jmix/quartzflowui/view/jobs/JobModelListView.java

```diff
@@ -129,16 +142,71 @@ private String getHeaderForPropertyColumn(String propertyName) {
         return messageTools.getPropertyCaption(jobModelsDc.getEntityMetaClass(), propertyName);
     }
 
+    protected void initDataChangedEventListener() {
+        VaadinSession session = VaadinSession.getCurrent();
+        if (session == null) {
+            return;
+        }
+
+        UiEventsManager uiEventsManager = session.getAttribute(UiEventsManager.class);
+        if (uiEventsManager != null) {
+            uiEventsManager.addApplicationListener(this, this::onApplicationEvent);
+
+            // Remove on detach event
+            addDetachListener(event -> uiEventsManager.removeApplicationListeners(this));
+        }
+    }
+
+    protected void onApplicationEvent(ApplicationEvent event) {
+        if (event instanceof QuartzDataChangedEvent dataChangedEvent) {
+            onDataChangedEvent(dataChangedEvent);
+        }
+    }
+
+    protected void onDataChangedEvent(QuartzDataChangedEvent event) {
+        loadJobsData();
```

Comment:
Do not reload all data every time - update target items in grid

Reply:
switched to ds item modification